cmd/pj-rehearse/server.go

```diff
@@ -149,6 +149,14 @@ func serverFromOptions(o options) (*server, error) {
 	}
 	rehearsalConfig.QuayIOImageHelper = quayIOImageHelper
 
+	if o.supplementalImagesConfigPath != "" {
+		if c, err := quayiociimagesdistributor.LoadConfig(o.supplementalImagesConfigPath); err != nil {
+			return nil, fmt.Errorf("failed to load config file %s: %w", o.supplementalImagesConfigPath, err)
+		} else {
+			rehearsalConfig.SupplementalCIImagesConfig = c
```

Comment:
We could just save and pass the targets of the mirroring (the keys of the mapping) onto the function.
We may call it as `ignoredTargets` (with type `sets[string]`) or something.